graphql/resolvers.go

```diff
@@ -50,6 +50,27 @@ func (r *mutationResolver) AddFavoriteProject(ctx context.Context, identifier st
 	}, nil
 }
 
+func (r *mutationResolver) RemoveFavoriteProject(ctx context.Context, identifier string) (*restModel.UIProjectFields, error) {
+	p, err := model.FindOneProjectRef(identifier)
+	if err != nil || p == nil {
+		return nil, errors.Errorf("could not find project '%s'", identifier)
+	}
+
+	usr := route.MustHaveUser(ctx)
+
+	err = usr.RemoveFavoriteProject(identifier)
+	if err != nil {
+		return nil, errors.Wrap(err, "error removing project from user's favorites")
+	}
+
+	return &restModel.UIProjectFields{
```

Comment:
isn't there a converter here?

Reply:
i don't think so

Reply:
that seems like a thing that we should do. having seperate data conversion code from the route implementation seems useful.

Reply:
can you help me to understand why using this model directly is not sufficient? @john-m-liu thinks it is

Reply:
there's no problem with this, though I think it's clearer to keep the conversion seperate from the business logic implementation, and if you ever have another resolover that produces this data type, then you'll end up needing to write the converter a second time, so it seems like a reasonable place to start